<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionRecognitionProjectClass</class>
 <widget class="QMainWindow" name="ActionRecognitionProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActionRecognitionProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="play_pause_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_sequence">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="output_sequence">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame_label">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>320</y>
      <width>50</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="convertMoSIFTToMoFREAK">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert MoSIFT to MoFREAK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train_svm_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Train SVM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_svm_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test SVM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leave_one_out_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test SVM with Leave-One-Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cluster_push_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cluster MoSIFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cluster_mofreak_push_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cluster MoFREAK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionLoadMoSIFT"/>
    <addaction name="actionLoadEverything"/>
    <addaction name="separator"/>
    <addaction name="actionLoadTrainingFile"/>
    <addaction name="separator"/>
    <addaction name="actionLoadMoSIFTForClustering"/>
    <addaction name="actionLoadMoFREAKForClustering"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load Video</string>
   </property>
  </action>
  <action name="actionLoadMoSIFT">
   <property name="text">
    <string>Load MoSIFT Features</string>
   </property>
  </action>
  <action name="actionLoadEverything">
   <property name="text">
    <string>Load Group of MoSIFT Features and AVI Files</string>
   </property>
  </action>
  <action name="actionLoadTrainingFile">
   <property name="text">
    <string>Load SVM Training File</string>
   </property>
  </action>
  <action name="actionLoadTestingFile">
   <property name="text">
    <string>Load SVM Testing File</string>
   </property>
  </action>
  <action name="actionLoadMoSIFTForClustering">
   <property name="text">
    <string>Load MoSIFT Files for Clustering</string>
   </property>
  </action>
  <action name="actionLoadMoFREAKForClustering">
   <property name="text">
    <string>Load MoFREAK Files for Clustering</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ActionRecognitionProject.qrc"/>
 </resources>
 <connections/>
</ui>
